iglu-registry/README.md

```diff
@@ -0,0 +1,145 @@
+# Iglu Registry
+
+## Introduction
+
+This folder contains
+
+* a docker compose example to run [Iglu Registry](iglu-registry) with [Postgres](https://github.com/docker-library/postgres)
+* docker images of [Iglu Registry](iglu-registry)
+  * `0.2.0`
+  * `0.3.0`
+
+
+[Iglu Registry](iglu-registry) images are based on [Snowplow base image][base-image] which leverages [the Java 8 Alpine image][alpine-image].
+
+## Usage
+
+Starting iglu registry is handed to docker-compose since it needs a postgres instance to function.
+
+Below are steps to get started with Iglu Registry:
+  1) Build iglu-registry image
+  2) Run docker-compose
+  3) Insert super api key to database
+
+After these steps, you can start using Iglu Registry through either Swagger UI on `localhost:8080` or any HTTP client (e.g. cURL, Postman)
+
+### Build
+
+1) Clone repository
+
+    `$ git clone git@github.com:snowplow/snowplow-docker.git`
+
+2) Go into `iglu-registry` folder
+
+    `$ cd iglu-registry`
+
+3) Build image    
+
+    `$ docker build -t iglu-registry iglu-registry/<registry_version>`
+
+    where `<registry_version>` is version of Iglu Registry one wants to run. Each version of Iglu Registry has a dedicated folder under `iglu-registry/` which contains `Dockerfile`
+
+### Run
+
+`Iglu Registry` makes use of an UUID key, called super API key, to generate other API keys.
+
+Before running, make sure to replace the API_KEY environment variable in `iglu-registry/docker-compose.yml` with your preferred super API key. e.g. 1d9c7e70-012b-11e8-ba89-0ed5f89f718b
+
+Finally, run Iglu Registry with Postgres in detached mode
+
+```
+$ docker-compose up -d
+```
+
+### Insert the super API key
+
+The following command inserts the super API key which can be used for generating API keys with read/write privileges for a specific vendor prefix.
+
+```
+ $ docker exec iglu-registry /snowplow/bin/super_api_key_init.sh
```

Comment:
why does the user need to do this? can't this be part of the container setup

Reply:
This script inserts super API key, provided in `docker-compose.yml` , to `apikeys` table in postgres. I don't think iglu-registry container should run this command without being sure about postgres, which docker-compose provides.